### ISSUE TITLE
 Not all CSV report files are generated after clicking CSV Report button

### ISSUE THREAD

**Issue by ArekKuczynski** [damies13/rfswarm#254]
**Describe the bug**
When you click CSV Report button only once, you will get only a summary file. However, when you click it a second time you will get all 3 files this time.

**To Reproduce**
1. Open Manager
2. Open Agent
3. Run some test
4. When test is finished click CSV Report button
5. Check result directory and see that there is only one csv file there
6. Click CSV Report button second time
7. Check result directory and now there should be every csv file generated

**Expected behavior**
All 3 csv files should be generated after just a single click of CSV Report button

**Screenshots**
After one click:
![image](https://github.com/damies13/rfswarm/assets/141224434/b6878643-48f6-48a2-82f5-37887edb3d33)
After second click:
![image](https://github.com/damies13/rfswarm/assets/141224434/9bb9b851-fc85-4dfa-afeb-9a327a951dac)

**Desktop (please complete the following information):**
- OS: Windows
- Version: 1.3.1

**Comment by damies13** [comment]
@ArekKuczynski can you create a test case for this? I'll add it to 1.3.2

**Comment by ArekKuczynski** [comment]
I almost made a test case for Issue #190 it also checks the number of csv files

**Comment by ArekKuczynski** [comment]
Should i make a pull request to 1.3.2 or 1.4.0?

**Comment by damies13** [comment]
They should be 
- Test cases for #190 should be PR to v1.4.0
- Test case for this issue as a PR for v1.3.2

For this issue you can use the same test case as `The results export button works in the manager` on the 1.3.2 branch just have the 1 issue tag on the 1.4.0 branch it will have tags for all issue numbers

**Comment by damies13** [comment]
Fix merged into v1.3.2 branch
